.coveragerc

```
@@ -4,7 +4,7 @@ skip_covered = false
 
 [paths]
 source =
-  src/propcache
+  src
   */lib/pypy*/site-packages/propcache
   */lib/python*/site-packages/propcache
   */Lib/site-packages/propcache
```

This entire section is not inclusion but simply a path mapping definition
```suggestion
  */lib/pypy*/site-packages
  */lib/python*/site-packages
  */Lib/site-packages
```